spec/system/publishers_can_edit_a_draft_vacancy_as_a_school_spec.rb

```diff
@@ -44,13 +45,23 @@
     end
 
     describe "submitting a completed draft" do
-      scenario "can submit a completed draft from the manage job listing page" do
-        visit organisation_job_path(complete_vacancy.id)
+      let(:completed_steps) do
+        %w[job_role job_details pay_package working_patterns important_dates documents applying_for_the_job job_summary]
+      end
 
+      scenario "can submit a completed draft from the manage job listing page" do
         click_on I18n.t("buttons.submit_job_listing")
 
         expect(page.current_path).to eq(organisation_job_summary_path(complete_vacancy.id))
       end
+
+      scenario "correct status tag is displayed" do
```

Comment:
I don't think this quite fits in the describe "submitting a completed draft" because that doesn't happen in this scenario, not sure what to rename it to though

Reply:
Made some changes. Take a look!